src/settings/index.ts

```diff
@@ -505,7 +506,11 @@ export const FIREBASE_CONFIG = {
   storageBucket: BUCKET_URL,
 };
 
-export const DEFAULT_SPACE_SLUG = "bootstrap";
+export const DEFAULT_SPACE_SLUG = "bootstrap" as SpaceSlug;
+export const DEFAULT_WORLD_SLUG = "bootstrap" as WorldSlug;
+// @debt This shouldn't be needed
+export const EMPTY_WORLD_SLUG = "" as WorldSlug;
+export const EMPTY_SPACE_SLUG = "" as SpaceSlug;
```

Comment:
I kept these so that it would be easier for me to find these pieces later. I would prefer that we don't have to worry about slugs being optionally passed *at all*. The URL function doesn't need to be smarter. Higher up the stack needs to be smarter. If we don't have a space slug by the time we get to these functions we're kind of screwed.

Reply:
In the most recent version I've deleted these constants entirely.

Regarding why bother with URL functions at all:
If we call `generatePath` throughout the codebase instead we have little to no validation on all of the parameters actually being passed in which will lead to even more mess.